bugbug/similarity.py

```diff
@@ -106,6 +114,28 @@ def get_text(self, bug, all_comments=False):
 
         return "{} {}".format(bug["summary"], comments)
 
+    def get_similar_bugs(self, bug):
+        similar_bug_ids = self.search_similar_bugs(bug)
+        if self.end_to_end:
+            similar_bugs = [
+                bug for bug in bugzilla.get_bugs() if bug["id"] in similar_bug_ids
+            ]
+            bug_couples = [(bug, similar_bug) for similar_bug in similar_bugs]
+            probs = list(
+                zip(
+                    self.duplicatemodel.classify(bug_couples, probabilities=True),
+                    bug_couples,
+                )
+            )
+            probs.sort(key=lambda v: -v[0][1])
```

Comment:
Nit: instead of converting to list and then using sort, you can use sorted:
```
probs_bugs = zip(...)
probs_bugs = sorted(probs_bugs, ...)
```
or even:
```
probs_bugs = sorted(zip(...), ...)
```